react/BellIcon/BellIcon.js

```diff
@@ -0,0 +1,11 @@
+import markup from './BellIcon.svg';
+import PropTypes from 'prop-types';
```

Comment:
PropTypes does not seem to be used here.

Reply:
Removed